echoprometheus/prometheus.go

```diff
@@ -78,10 +79,29 @@ type MiddlewareConfig struct {
 	// If DoNotUseRequestPathFor404 is true, all 404 responses (due to non-matching route) will have the same `url` label and
 	// thus won't generate new metrics.
 	DoNotUseRequestPathFor404 bool
+
+	// StatusCodeResolver resolves err & context into http status code. Default is to use context.Response().Status
+	StatusCodeResolver StatusCodeResolveFunc
 }
 
 type LabelValueFunc func(c echo.Context, err error) string
 
+type StatusCodeResolveFunc func(c echo.Context, err error) int
```

Comment:
lets not define it as type. It add another public API contact we need to maintain. I think field declaration `StatusCodeResolver func(c echo.Context, err error) string` would be fine.

Reply:
OK, I'll fix it